test/jdk/java/lang/StringBuffer/IndexOf.java

```diff
@@ -31,158 +31,280 @@
 
 public class IndexOf {
 
-    static Random generator = new Random();
-    private static boolean failure = false;
+  static Random generator = new Random(1999);
+  private static boolean failure = false;
 
-    public static void main(String[] args) throws Exception {
-        simpleTest();
-        compareIndexOfLastIndexOf();
-        compareStringStringBuffer();
+  public static void main(String[] args) throws Exception {
+    String testName = "IndexOf";
 
-        if (failure)
-           throw new RuntimeException("One or more BitSet failures.");
+    for (int i = 0; i < 20000; i++) {
+      int foo = testName.indexOf("dex");
     }
-
-    private static void report(String testName, int failCount) {
-        System.err.println(testName+": " +
-                         (failCount==0 ? "Passed":"Failed("+failCount+")"));
-        if (failCount > 0)
-            failure = true;
+    System.out.println("");
+    generator.setSeed(1999);
```

Comment:
Is there a good reason for a fixed seed?

Reply:
Nope :-). Needed consistency during testing.